app/main.py

```diff
@@ -1,6 +1,19 @@
 from typing import Callable
+from functools import wraps
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    results = {}
+
+    @wraps(func)
+    def wrapper(*args, **kwargs) -> int:
```

Comment:
Please, indicate that `wrapper` could return `Any` instead of `int` in the return type annotation